mod/gcforums/views/default/forms/gcforums/edit.php

```diff
@@ -1,6 +1,8 @@
 <?php
 
 //elgg_push_breadcrumb('step 1');
```

Comment:
Is this comment related to the code you inserted below? otherwise it might be confusing